test/testdrive/mzcompose.py

```diff
@@ -55,6 +55,13 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         default=Materialized.Size.DEFAULT_SIZE,
         help="Use SIZE 'N-N' for replicas and SIZE 'N' for sources",
     )
+    parser.add_argument(
+        "--dyncfg",
+        type=str,
+        action="append",
+        nargs="*",
+        help="Dyncfgs to enable in Materialize",
```

Comment:
This looks like it would support any system parameter, so I'd be tempted to name it more generally!

Reply:
Good call, updated!